src/PortingAssistant.Client.Client/PortingAssistantClient.cs

```diff
@@ -68,6 +68,9 @@ public async Task<SolutionAnalysisResult> AnalyzeSolutionAsync(string solutionFi
                 {
                     SolutionName = Path.GetFileNameWithoutExtension(solutionFilePath),
                     SolutionFilePath = solutionFilePath,
+                    SolutionGuid = FileParser.SolutionFileParser.getSolutionGuid(solutionFilePath),
+                    RepositoryUrl = FileParser.GitConfigFileParser.getGitRepositoryUrl(
```

Comment:
this code will throw error if FileParser.GitConfigFileParser.getGitRepositoryRootPath(solutionFilePath) return null, getGitRepositoryUrl fail

Reply:
Thank you for reveiwing @saikiranakula-amzn, `getGitRepositoryUrl` handles the scenario when input is `null`. Particularly, when it's checking if directory exists: `if (!Directory.Exists(gitRepositoryRootPath)) {return null;}`, if the input `gitRepositoryRootPath` is null, `Directory.Exists` returns `false`, so `null` will be returned. There is a [unit test](https://github.com/aws/porting-assistant-dotnet-client/pull/171/files?file-filters%5B%5D=.cs#diff-fda9fd488e7c765eed353f8060241944e5cd502e46be4fd21b49daaa8599b951R42) that covers this case.